web/src/app/users/util.test.js

```diff
@@ -6,12 +6,12 @@ import {
 } from './util'
 
 test('formatPhoneNumber', () => {
-  expect(formatPhoneNumber('+17635550100')).toBe('+1 763-555-0100')
-  expect(formatPhoneNumber('+12085550105')).toBe('+1 208-555-0105')
-  expect(formatPhoneNumber('+15165550184')).toBe('+1 516-555-0184')
-  expect(formatPhoneNumber('+911400000000')).toBe('+91 14 0000 0000')
-  expect(formatPhoneNumber('+911401234567')).toBe('+91 14 0123 4567')
-  expect(formatPhoneNumber('+911409876543')).toBe('+91 14 0987 6543')
+  expect(formatPhoneNumber('+17635550100')).toBe('+1 (763) 555-0100')
```

Comment:
International format should be `+1 763-555-0123`

![image](https://user-images.githubusercontent.com/595010/61238304-fcd2f380-a701-11e9-866f-8480ac6c9e79.png)

Reply:
This is a US format though, isn't it? We only support US, India, and the UK. We will be deleting and replacing these tests once we support international phone numbers from previous discussions we've had.

Reply:
It's not the format outside of the US though. It would be incorrectly formatted for users in the UK or India. That's why we format all numbers as international, as it is the same in and from all locations, everyone sees the same standard format.

For example, the national format of an India number is `123 123 1234` which isn't used in the US.

![image](https://user-images.githubusercontent.com/595010/61242216-bd5cd500-a70a-11e9-9c7a-f255a9ede973.png)

And in any case, if we change it here (thus in the cypress tests) we will just be changing it back in a few days when the CC PR lands as formatting will happen by `libphonenumber`.

Reply:
Sure.